schema.sql

```diff
@@ -69,7 +69,11 @@ CREATE TABLE genotypes (
        quality TEXT,
 
        -- Cycledash-derived data
-       "annotations:gene_names" TEXT,
+       -- These are from the varcode annotation worker
+       "annotations:gene_name" TEXT,
```

Comment:
Hmm—is there a way to get that back? That seems like a useful feature /cc @iskandr @tavinathanson